shared/modules/hexstring-utils.ts

```diff
@@ -66,6 +66,16 @@ export function isValidHexAddress(
   return isValidAddress(addressToCheck);
 }
 
+/**
+ * Determines if a string is a possible ethereum address
+ *
+ * @param candidate - the input to check
+ * @returns true if the input is a 40 char hex string with optional 0x prefix, false otherwise
```

Comment:
```suggestion
 * Determine if a string is a possible Ethereum address.
 *
 * @param candidate - The input to check.
 * @returns `true` if the input is a 40-character-long hex string with optional 0x-prefix, `false` otherwise.
```